### PR TITLE
Pongo editor fixes canary

### PR DESCRIPTION
**Pull request by frimpongopoku**
Hi @BradHN1, cant say anything is fixed, because the editor was never broken here 🤣 🤣 . 
Even after working with the content in the canary database locally, everything still works fine. 

However, I made a tiny tweak in the editor. I Made the `handleEditorChange` function a synchronous function , since its  our old editor that needed that function to be `async` anyway. Locally, the editor still works with our without the change though. 

But since we are dealing with weird catches, lets redeploy to canary again, and see if there is a change.

### CODE REVIEW

File: v2.0/app/utils/messenger.js [hunk 1]
```diff
@@ -21,40 +21,42 @@ export async function apiCall(
...
-  if (destinationUrl.charAt(0) === '/') {
+  if (destinationUrl.charAt(0) === "/") {
     destinationUrl = destinationUrl.substring(1);
   }
 
-  if (IS_LOCAL) {
-    destinationUrl = "api/" + destinationUrl;
-  }
-  
+  // if (IS_LOCAL) {
```
Comment (BradHN1) [inline v2.0/app/utils/messenger.js:39]: This will break working locally.

Comment (frimpongopoku) [reply v2.0/app/utils/messenger.js:39]: No, not with Sam's new changes. Infact, admin-frontend does not work when that part is not commented out. None of the links to the backend work.

**Review by BradHN1 — CHANGES_REQUESTED** [review]
The lines "// if (IS_LOCAL) { ..." was a change I made so one could run the admin portal with a local API.  Please change that back unless you have another change which fixes that.
Also I suggest not merging  canary into a feature branch off of Dev, or branching off canary.  This can cause problems for merging.  Thanks

File: v2.0/app/utils/messenger.js (hunk 1 above)
Comment (BradHN1) [reply v2.0/app/utils/messenger.js:39]: That change was needed because of Sam's changes, and he approved this.  So I don't quite understand.

**Comment by frimpongopoku** [comment]
Wait, has the api structure been changed again?? Cos on my end, I had to comment that out to be able to work with the latest dev api changes `locally` as well. Why does yours work differently? Is there anything I need to pull from the backend?? If so from where?

**Pull request closed without merging**